third_party/pcsc-lite/naclport/server/src/public/pcsc_lite_server_web_port_service.h

```diff
@@ -42,14 +42,15 @@ namespace google_smart_card {
 //
 // Note: All methods except GetInstance are thread safe. Calls to GetInstance
 //       concurrent to class construction or destruction are not thread safe.
-class PcscLiteServerGlobal final {
+class PcscLiteServerWebPortService final {
```

Comment:
I'm only unsure about having both of words "server" and "service" within a single name.

Reply:
I think it is fine